daemons/controld/controld_throttle.c

```diff
@@ -409,14 +409,25 @@ static void
 throttle_update_job_max(const char *preference)
 {
     long long max = 0LL;
+
+    // Per-node override
     const char *env_limit = pcmk__env_option(PCMK__ENV_NODE_ACTION_LIMIT);
 
     if (env_limit != NULL) {
-        preference = env_limit; // Per-node override
+        int rc = pcmk__scan_ll(env_limit, &max, 0LL);
+
+        if (rc != pcmk_rc_ok) {
+            crm_warn("Ignoring local option PCMK_" PCMK__ENV_NODE_ACTION_LIMIT
+                     " because '%s' is not a valid value: %s",
+                     env_limit, pcmk_rc_str(rc));
+            env_limit = NULL;
```

Comment:
behavioral change, falling back to `preference` argument if parsing local option fails; probably fine

Reply:
yeah, i considered that a fix, but not worth calling out